src/prog_models/prognostics_model.py

```diff
@@ -435,11 +451,12 @@ def performance_metrics(self, x) -> dict:
 
         Example
         -------
-        | m = PrognosticsModel() # Replace with specific model being simulated
-        | u = m.InputContainer({'u1': 3.2})
-        | z = m.OutputContainer({'z1': 2.2})
-        | x = m.initialize(u, z) # Initialize first state
-        | pm = m.performance_metrics(x) # Returns {'tMax':33, 'iMax':19}
+        >>> from prog_models.models import BatteryElectroChemEOD
+        >>> m = BatteryElectroChemEOD() # Replace with specific model being simulated
+        >>> u = m.InputContainer({'u1': 3.2})
+        >>> z = m.OutputContainer({'z1': 2.2})
+        >>> x = m.initialize(u, z) # Initialize first state
+        >>> pm = m.performance_metrics(x)   # {'max_i': array([8.83810109])}
```

Comment:
No, since there is no return. You only do that if there is an expected value returned and that would be the value in the example. I left those comments to the side to explain what would be in the variable.